lib/pages/DesktopScout.dart

```diff
@@ -20,6 +20,9 @@ class _DesktopScoutState extends State<DesktopScout> {
   int sportState = 0;
   List<Athlete> nflList = [];
   List<Athlete> nflListFilter = [];
+  String tt = "All Sports";
```

Comment:
Can you make this string name meaningful?

Reply:
done